### DISCUSSION TITLE
How to style a third-party component with chakra's styles

### DISCUSSION THREAD

**Discussion by yantakus** [chakra-ui/chakra-ui#3688] [General] [answered]
What if I want to use a third-party component, let's say react-select and I want to style it with chakra's styles. I mean with my custom theme that extends chakra's default theme and ovverrides some of its styles.

1) Can I somehow access compiled chakra styles, like I do it with tailwind, something like `<Select className="border rounded-lg ..." />`.
2) Or at least style it with styled-components. something like this:
```
import { theme } from 'chakra-ui'
import Select from 'react-select'
import styled from 'styled-components'

const StyledSelect = styled(Select)`
  .select-icon {
    color: ${theme.colors.red[600]};
  }
`
```

Thanks.

**Comment by malashevskyi** [comment, marked as answer]
Hi, you need to put your third-party component in `as={}` attribute.
```javascript
<Box as={Select} maxW="400px" ...otherProps />
```
Then you will be able to control styles of a component container with chakra styles. If you want to use chakra styles for children of a component then you can use react-select configuration and `useTheme` chakra hook. 
[example](https://codesandbox.io/s/chakra-react-select-ssxwy)
